kotlin/lib/src/main/kotlin/EventTypeListOptions.kt

```diff
@@ -0,0 +1,3 @@
+package com.svix.kotlin
+
+class EventTypeListOptions(var withContent: Boolean = false) : ListOptions()
```

Comment:
Put this in the EventType file? (same for the rest of them)

Reply:
Done.